kevm-pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -42,9 +57,19 @@ def __init__(self, name: str, id: int, abi: Dict, contract_name: str, sort: KSor
             # TODO: Check that we're handling all state mutability cases
             self.payable = abi['stateMutability'] == 'payable'
 
+        @property
+        def signature(self) -> str:
+            arg_list = ','.join(self.arg_types)
+            return f'{self.name}({arg_list})'
+
+        @property
+        def klabel(self) -> KLabel:
+            args_list = '_'.join([_evm_base_sort(input_type).name for input_type in self.arg_types])
```

Comment:
```suggestion
            args_list = '_'.join(_evm_base_sort(input_type).name for input_type in self.arg_types)
```

Reply:
So instead of using `_evm_base_sort(input_type).name`, I think just use `input_type` here.